src/parser/warrior/arms/modules/core/Slam.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import { formatPercentage } from 'common/format';
+import SpellLink from 'common/SpellLink';
+import SPELLS from 'common/SPELLS';
+import Analyzer, { SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import SpellUsable from 'parser/warrior/arms/modules/features/SpellUsable';
+
+class Slam extends Analyzer {
+  static dependencies = {
+    spellUsable: SpellUsable,
+  };
+
+  badCast = 0;
+  totalCast = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.addEventListener(Events.cast.by(SELECTED_PLAYER).spell(SPELLS.SLAM), this._onSlamCast);
+  }
+
+  _onSlamCast() {
+    this.totalCast += 1;
+    if (this.spellUsable.isAvailable(SPELLS.MORTAL_STRIKE.id)) {
+      this.badCast += 1;
+    }
+  }
+
+  get badCastSuggestionThresholds() {
+    return {
+      actual: this.badCast / this.totalCast ,
```

Comment:
You should probably check for division by zero here
```suggestion
      actual: (this.badCast / this.totalCast) || 0,
```